tests/robustness/failpoint/cluster.go

```diff
@@ -40,6 +40,10 @@ var (
 	MemberReplace          Failpoint = memberReplace{}
 	MemberDowngrade        Failpoint = memberDowngrade{}
 	MemberDowngradeUpgrade Failpoint = memberDowngradeUpgrade{}
+	GrowMember             Failpoint = &growMember{
```

Comment:
We are not growing members here :P 

Usually talk about "cluster" and not "members". And we call the process "scale up". Maybe `ClusterScaleUp`?

Reply:
How about `ClusterScaleInAndOut` because we're doing both? :)